examples/foo-missionmodel/src/test/resources/gov/nasa/jpl/aerie/foomissionmodel/plan.json

```diff
@@ -2,24 +2,29 @@
     "type": "foo",
     "arguments":
       { "x": 1,
-        "y": "test_1" } },
+        "y": "test_1",
+        "z": 4 } },
```

Comment:
Quick sanity check: where did the new `z` argument come from?

Reply:
Good question :+1: It came from #334, when we wanted to include a "required" parameter in our sample mission models - since we had a regression where required parameters would fail to instantiate.

This plan.json is used in the main method in `SimulateMapSchedule` - it's not used for any automated tests, but it can be used for manual testing. We should have added the z parameter in #334. 

It's unrelated to this PR - so I would accept the argument that it should be split into its own PR :+1: But it is such a small, inconsequential change, that I'd also be okay with bundling it into this PR.

Reply:
It's in its own commit, so I think it's fine. IMHO, we should feel comfortable cleaning up things as we come in contact with code, even if it's technically not required for the task at hand.

Reply:
It would be / would have been good to include that context in the commit message. (Remember, the title needs to be short, but you can have as long a commit body as you desire.)